src/sentry/utils/outcomes.py

```diff
@@ -135,26 +64,6 @@ def track_outcome(
 
     timestamp = timestamp or to_datetime(time.time())
 
-    tsdb_in_consumer = decide_tsdb_in_consumer()
-
-    if not tsdb_in_consumer:
-        increment_list = list(
-            tsdb_increments_from_outcome(
-                org_id=org_id,
-                project_id=project_id,
-                key_id=key_id,
-                outcome=outcome,
-                reason=reason,
-                category=category,
-            )
-        )
-
-        if increment_list:
-            tsdb.incr_multi(increment_list, timestamp=timestamp)
-
-        if project_id and event_id:
-            mark_tsdb_incremented(project_id, event_id)
```

Comment:
Did you verify none, except for the consumer that is supposed to, actually depends on this cache ?

Reply:
yes